pubsub/api/Program.cs

```diff
@@ -0,0 +1,157 @@
+﻿/*
+ * Copyright (c) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace PubSubSample
+{
+
+  public class Program
+  {
+    public static void Main(string[] args)
+    {
+      if (args.Length == 0)
+        PrintUsage();
+      else
+        RunCommand(args.First(), args.Skip(1).ToArray());
+    }
+
+    static void PrintUsage()
+    {
+      Console.WriteLine(@"Usage: PubSubSample.exe [command] [args]

```

Comment:
That's interesting.  We've been steered in different directions because our documentation was different.  The BigQuery samples:
https://cloud.google.com/bigquery/docs/managing_jobs_datasets_projects
Do not include the using statements in any programming language.